src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -619,7 +619,7 @@ private PSModuleInfo ImportModule_LocallyViaName_WithTelemetry(ImportModuleOptio
                 // avoid double reporting for WinCompat modules that go through CommandDiscovery\AutoloadSpecifiedModule
                 if (!foundModule.IsWindowsPowerShellCompatModule)
                 {
-                    ApplicationInsightsTelemetry.SendTelemetryMetric(TelemetryType.ModuleLoad, foundModule.Name);
+                    ApplicationInsightsTelemetry.SendModuleTelemetryMetric(TelemetryType.ModuleLoad, foundModule.Name, foundModule.Version.ToString());
```

Comment:
Is there a case where Version is null like you cover on line 899 below?

Reply:
You use the null conditional operator in the other cases below, so it looks like an omission here?

Reply:
the PSModuleInfo structure does not have this member as nullable, it shouldn't need the protection